### ISSUE TITLE
Favicon and footer

### ISSUE THREAD

**Issue by Sonnysam** [KBismark/statestore.js#2]
Add a favicon, footer and more content more like learn more, the page is good but is less informative

**Comment by KBismark** [comment]
Yes. I'm glad you found time to go through and I appreciate the feedback. The page is current in the process of development. All these you suggest will be added soon.
